src/Storage/Storage.Management/Az.Storage.psd1

```diff
@@ -57,7 +57,17 @@ RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.3.0'; })
 
 # Assemblies that must be loaded prior to importing this module
 RequiredAssemblies = '.\Microsoft.Azure.Management.Storage.dll', 
-               '.\Microsoft.WindowsAzure.Storage.dll'
```

Comment:
We will need to move this dependency into profile

Reply:
Which dependenc do you mean? for "Microsoft.WindowsAzure.Storage.dll", it's already in Az.Accounts, and removed from Az.Storage with this PR.